.husky/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+if [ -t 1 ]; then
+  exec < /dev/tty
+fi
+
+npx lint-staged
```

Comment:
should be ˋyarn lint-stagedˋ, which will run much faster the first time because yarn already has it cached, since it's in the dev deps.

Reply:
Updated